services/system/Transact/plugin/src/lib.rs

```diff
@@ -166,7 +166,6 @@ impl Admin for TransactPlugin {
         if actions.len() == 0 {
             return Ok(());
         }
-        CurrentActions::clear();
```

Comment:
Why doesn't it need to be cleared?

Reply:
I think leaving it in is harmless and is more consistent and future-proof.

Reply:
On the contrary, I was planning to add an in-memory storage interface to `clientdata`, use it for all of this hook data in `transact`, and then remove all of the manual clears (and remove all associated checks in `start_tx`). 

If we have a working in-memory storage library, I don't think users should have to re-verify it